src/main/java/me/botsko/prism/utils/EnchantmentUtils.java

```diff
@@ -79,8 +80,13 @@ public static Enchantment getEnchantmentFromCommonName(String name) {
             return Enchantment.DAMAGE_UNDEAD;
         } else if (name.equalsIgnoreCase("unbreaking")) {
             return Enchantment.DURABILITY;
+        } else if (name.equals("vanishing curse")) {
```

Comment:
Codacy found an issue: [Position literals first in String comparisons](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5766072)